### ISSUE TITLE
Wrong autocorrect for `Style/OperatorMethodCall` with `foo./(bar)`

### ISSUE THREAD

**Issue by Earlopain** [rubocop/rubocop#13225]
## Expected behavior

```rb
foo./(bar)
# =>
foo /(bar)
```

This correction is not valid syntax. Note that all other operators are valid in that context.

I think the correction should instead be this:

```rb
# Added space makes this ok
foo / (bar)
```

```diff
diff --git a/spec/rubocop/cop/style/operator_method_call_spec.rb b/spec/rubocop/cop/style/operator_method_call_spec.rb
index 89ef96fc2..12501b584 100644
--- a/spec/rubocop/cop/style/operator_method_call_spec.rb
+++ b/spec/rubocop/cop/style/operator_method_call_spec.rb
@@ -60,7 +60,7 @@ RSpec.describe RuboCop::Cop::Style::OperatorMethodCall, :config do
 
       # Redundant parentheses in `(bar)` are left to `Style/RedundantParentheses` to fix.
       expect_correction(<<~RUBY)
-        foo #{operator_method}(bar)
+        foo #{operator_method} (bar)
       RUBY
     end
```

However, I could not figure out how to make autocorrect add this whitespace properly. Related to #13224

## RuboCop version

```
$ bundle exec rubocop -V
1.66.1 (using Parser 3.3.4.2, rubocop-ast 1.32.3, running on ruby 3.3.5) [x86_64-linux]
  - rubocop-performance 1.21.1
  - rubocop-rake 0.6.0
  - rubocop-rspec 3.0.4
```
